hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -482,3 +506,424 @@ def hpat_pandas_series_floordiv_number_impl(lhs, rhs):
         return hpat_pandas_series_floordiv_number_impl
 
     raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+
+
+@overload_method(SeriesType, 'pow')
+def hpat_pandas_series_pow(lhs, rhs, level=None, fill_value=None, axis=0):
+    """
+    Pandas Series method :meth:`pandas.Series.pow` implementation.
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5
+
+    Parameters
```

Comment:
Please add description for parameter "rhs" here and for other implementations in this commit.

Reply:
Nice catch!
Also, please use supported types list as it presented in example here https://github.com/IntelPython/hpat/blob/master/hpat/datatypes/hpat_pandas_series_functions.py#L57